packages/react-aria-components/docs/Table.mdx

```diff
@@ -499,13 +499,15 @@ A `Column` can be targeted with the `.react-aria-Column` CSS selector, or by ove
 
 ### TableBody
 
-A `TableBody` can be targeted with the `.react-aria-TableBody` CSS selector, or by overriding with a custom `className`.
+A `TableBody` can be targeted with the `.react-aria-TableBody` CSS selector, or by overriding with a custom `className`. It supports the following states and render props:
```

Comment:
Mega nit: I see changes from `support` to `provide` and vice versa. Quite frankly I think having both is fine, but it seems to be the bulk of the changes so just wanna make sure we weren't trying to only use `support` in these blurbs. Or maybe there is some other reasoning to using each in various situations

Reply:
There wasn't a specific reason for this, we just needed `and render props` added in lots of places. I can review and make sure these are all the same. Not a big deal which we use, I think I prefer `provides` is a little more.

Reply:
Went ahead and reverted a bunch of these changes for simplicity. I think it was fine as-is

Reply:
TableBody is only showing the state `isEmpty`, should we edit that sentence or are there missing props?